app/models/wizards/steps/choose_sektion_form.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module Wizards
+  module Steps
+    class ChooseSektionForm < Step
```

Comment:
Should the model also contain `Form` in its name? I find it a bit confusing.
```suggestion
    class ChooseSektion < Step
```